### DISCUSSION TITLE
Enter to send

### DISCUSSION THREAD

**Discussion by Vinay-052** [open-webui/open-webui#5077] [General]
I've been using shift+enter to enter a new line and would love to disable this.  

If am writing the content myself rather than pasting it from another source, it annoys that hitting enter will send the chat to model. I'd have to remember using shift at every line and sometimes this breaks concentration on what I am doing. 

It would be nice to have the option to **disable** enter to send and only use the send button next to the chat bubble. Checked through the interface options and surprised I couldn't find anything similar.

**Comment by jdeepwell** [comment]
Or, as in many other AI chat UIs, use Cmd/Ctrl Return to send and make (naked) Return just a new-line.

**Comment by beeender** [comment]
https://github.com/open-webui/open-webui/pull/10987

This has been implemented already.
